meta-lmp-bsp/recipes-bsp/lmp-boot-firmware/lmp-boot-firmware.bbappend

```diff
@@ -1,5 +1,5 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
 PROVIDES:uz3eg-iocc-sec = "virtual/boot-bin"
-PV:uz3eg-iocc-sec = "8"
+LMP_BOOT_FIRMWARE_VERSION = "8"
```

Comment:
Since LMP_BOOT_FIRMWARE_VERSION is global this would need to be set in meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc instead (LMP_BOOT_FIRMWARE_VERSION:uz3eg-iocc-sec).

Otherwise the value will not propagate over the other recipes/classes.

Reply:
fixed, thanks!